docs/querying/basics.md

```diff
@@ -249,11 +280,11 @@ The same works for range vectors. This returns the 5-minute rate that
 
     rate(http_requests_total[5m] @ 1609746000)
 
-The `@` modifier supports all representation of float literals described
-above within the limits of `int64`. It can also be used along
-with the `offset` modifier where the offset is applied relative to the `@`
-modifier time irrespective of which modifier is written first.
-These 2 queries will produce the same result.
+The `@` modifier supports all representations of numeric literals described above.
+It works with the `offset` modifier where the offset is applied relative to the `@`
```

Comment:
"works" implies it is typical to combine them, whereas "can be used" says it is possible but not the norm.
In this case I think "can be used" is better.

Also I think a new paragraph at "It" would help to separate the description of `@` from the combination of `@` and `offset` which follows.